### ISSUE TITLE
new plot function: error bars & timeseries plot

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/infographiq#10]
some time series csvs have standard error columns that could be added to the plots as [upper/lower bars](https://rstudio.github.io/dygraphs/gallery-upper-lower-bars.html).

**Comment by 7yl4r** [comment]
@eqmh says : error bars would make plots too busy & some wouldn't understand them.

Marking as wontfix for now.

**Comment by bbest** [comment]
Yes, we can make this an option in the [`plot_timeseries()`](https://github.com/marinebon/infographiq/blob/master/R/plot_timeseries.R) function for both types of variation:

1. Variation for each data point in time: show with **shading** a la [Upper/Lower Bars](http://rstudio.github.io/dygraphs/gallery-upper-lower-bars.html):
  
  ![image](https://user-images.githubusercontent.com/2837257/30559834-dcfcb63e-9c6a-11e7-85ca-fc47aed44d61.png)

1. Long-term deviation from historical mean / sd to flag anomolously high/low values: show as **horizontal bars**:
  
  ![image](https://user-images.githubusercontent.com/2837257/30559871-f3002dc6-9c6a-11e7-81ae-bfa7fb2643ce.png)

TODO: make clear in documentation.

**Comment by 7yl4r** [comment]
Now that I've hammered out how to handle multiple plotting functions, I think this would be best done as a separate plotting function. This might be as simple as copying over that example code (with modifications to use plot_indicators column names) into a `.rmd.brew` template in `./inst/site_template/plotting_functions`

**Comment by 7yl4r** [comment]
Turns out creating a new plotting function would mean duplicating a lot of code from `plot_dygraph_timeseries.R` so I added the std_err argument to the plotting function and implemented the error bars. IMO this function needs to be cleaned up and broken out into multiple functions that could be reused elsewhere.

![image](https://user-images.githubusercontent.com/1051390/31789666-6584d3a6-b502-11e7-93fd-2eaacd622320.png)

To make it work in info-fk all we need to do is change the column name in plot_indicators.csv from  `col_se` to `std_err`.
